package.json

```diff
@@ -107,6 +108,7 @@
     "strip-indent": "^2.0.0",
     "tap": "^12.6.1",
     "which": "^1.3.1",
+    "yaml": "^1.4.0",
```

Comment:
No point installing `yaml` dev dependency, it'll never be tested (js-yaml is found first).